### PR TITLE
Add init for Spring Boot on Camunda 7.13

### PR DESCRIPTION
**Pull request by falko**

### CODE REVIEW

**Comment by falko** [comment]
A test case would be good to write. But it would have to be skipped for older Camunda versions.

**Comment by zambrovski** [comment]
Do you think we can use @martinschimak `Feature` for this?

File: core/src/main/java/org/camunda/bpm/extension/process_test_coverage/junit/rules/ProcessCoverageConfigurator.java [hunk 1]
```diff
@@ -40,6 +50,25 @@ private static void initializeFlowNodeHandler(ProcessEngineConfigurationImpl con
...
+              .getDeclaredMethod("setCustomHistoryEventHandlers", List.class);
+          if (setCustomHistoryEventHandlers != null) {
+            setCustomHistoryEventHandlers.invoke(configuration, Arrays.asList((HistoryEventHandler) historyEventHandler));
+            Method setEnableDefaultDbHistoryEventHandler = ProcessEngineConfigurationImpl.class
+                .getDeclaredMethod("setEnableDefaultDbHistoryEventHandler", boolean.class);
+            if (setEnableDefaultDbHistoryEventHandler != null) {
+              setEnableDefaultDbHistoryEventHandler.invoke(configuration, false);
+            }
+          }
+        } catch (NoSuchMethodException | IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
```
[the remark this answers is not in the document]
Comment (falko) [reply core/src/main/java/org/camunda/bpm/extension/process_test_coverage/junit/rules/ProcessCoverageConfigurator.java:66]: Sure. Sound like a good idea. I didn't know about that [API util class](https://github.com/camunda/camunda-bpm-process-test-coverage/blob/master/core/src/main/java/org/camunda/bpm/extension/process_test_coverage/util/Api.java). Do you know how to use it here? Feel free to modify the PR. As a project member, you should have permissions.

**Comment by zambrovski** [comment]
Hi @falko 

I implemented the "feature" support and replaced your own implementation by it. I tested it locally using different profiles. Still not sure how you want to write a dedicated test for this. In both cases (prior 7.13 and after it) the springboot test seems to work (let us check the compatibility test in github actions for seure).

So from my point of view this is "good enogh". If you like more tests - feel free to contribute..

**Review by zambrovski — APPROVED** [review]

**Comment by zambrovski** [comment]
I'll merge it in to get the functionality in 0.4.0

**Pull request merged**